src/cascadia/TerminalApp/Utils.cpp

```diff
@@ -16,33 +16,3 @@ std::wstring GetWstringFromJson(const Json::Value& json)
 {

```

Comment:
Since we just moved `GetColoredIcon()` to the header file, why not move this one too? One less file to worry about.

Reply:
i don't hate it. I'd prefer to do that in a separate PR (and make it templatable :P)